Tools/autotest/sim_vehicle.py

```diff
@@ -229,154 +283,173 @@ def wait_unlimited():
 default_params_filename: filename of default parameters file.  Taken to be relative to autotest dir.
 extra_mavlink_cmds: extra parameters that will be passed to mavproxy
 """
-_options_for_frame = {
-    "calibration": {
-        "extra_mavlink_cmds": "module load sitl_calibration;",
-    },
-    # COPTER
-    "+": {
-        "waf_target": "bin/arducopter",
-        "default_params_filename": "default_params/copter.parm",
-    },
-    "quad": {
-        "model": "+",
-        "waf_target": "bin/arducopter",
-        "default_params_filename": "default_params/copter.parm",
-    },
-    "X": {
-        "waf_target": "bin/arducopter",
-        # this param set FRAME doesn't actually work because mavproxy
-        # won't set a parameter unless it knows of it, and the param fetch happens asynchronously
-        "default_params_filename": "default_params/copter.parm",
-        "extra_mavlink_cmds": "param fetch frame; param set FRAME 1;",
-    },
-    "hexa": {
-        "make_target": "sitl",
-        "waf_target": "bin/arducopter",
-        "default_params_filename": "default_params/copter.parm",
-    },
-    "octa-quad": {
-        "make_target": "sitl",
-        "waf_target": "bin/arducopter",
+_options = {
+    "ArduCopter": {
+        "frames": {
+            # COPTER
+            "+": {
+                "waf_target": "bin/arducopter",
+                "default_params_filename": "default_params/copter.parm",
+            },
+            "quad": {
+                "model": "+",
+                "waf_target": "bin/arducopter",
+                "default_params_filename": "default_params/copter.parm",
+            },
+            "X": {
+                "waf_target": "bin/arducopter",
+                # this param set FRAME doesn't actually work because mavproxy
+                # won't set a parameter unless it knows of it, and the param fetch happens asynchronously
         "default_params_filename": "default_params/copter.parm",
```

Comment:
Missing indent, and the comment above should really be above *extra_mavlink_cmds* (and by the way, the issue with setting parameters could be solved by using an option to SITL instead of via MAVProxy).